website_sale_wishlist/models/wishlist.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2016 Leonardo Donelli
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp import api, fields, models
+
+
+class ShopWishList(models.Model):
+    _name = 'wishlist'
```

Comment:
IMO this should be `product.wishlist` & `product.wishlist.item` - similar to `product.pricelist` and `product.pricelist.item`